dv/uvm/core_ibex/directed_tests/test_macros.h

```diff
@@ -0,0 +1,754 @@
+// See LICENSE for license details.
```

Comment:
As with `encoding.h` I think this has come from some other RISCV repo? The same comments apply in that this should really be vendored in rather than included directly here.